camsys/transam_spatial
# Transam dependencies

* Make travis get latest versions of depended-on engines
* Make sure Gemfile.lock is not committed
* Update schema to work with latest transam_core seeds
